code/WorkInProgress/Venseer/RemoteCyborg.dm

```diff
@@ -0,0 +1,304 @@
+* A verification about the camera, so it will not cause sparks
+* uplink_item.dm:
+* Added the box of items on the uplink_item.dm so the roboticist can pick the bundle to build a remote cyborg
+*
+*/
+
+var/global/list/rc_cameras = list();
+var/global/list/rc_control_boards = list();
+
+/*
+    START REMOTE CYBORG CAMERA SECTION
+*/
+/obj/item/device/syndicate_remote_cyborg_camera
+  name = "R.C. Camera"
+  desc = "A tiny camera with a small antenna sticking to its side. It is able to transmit a signal to anything that can accept it. Seems to blend well with complex machinery."
+  icon = 'icons/obj/RemoteCyborg.dmi'
+  icon_state = "flashing_camera"
+  w_class = W_CLASS_TINY
+  item_state = ""
+  throw_speed = 4
+  throw_range = 20
+  flags = NOBLUDGEON | FPRINT
+  var/frequency = "syndicate"
+  var/active = 0
+  var/mob/camera_target
+
+/obj/item/device/syndicate_remote_cyborg_camera/afterattack(atom/target, mob/user, proximity_flag)
+  if (proximity_flag != 1)
+    to_chat(user, "<span class='warning'>You are too far away to use this.</span>")
+    return
+  if(!istype(target, /mob/living/silicon/))
+    to_chat(user, "<span class='warning'>You can only apply this to cyborgs.</span>")
+    return
+  if(istype(target, /mob/living/silicon/ai))
+    to_chat(user, "<span class='warning'>You can not install this on an AI core.</span>")
+    return
+  if(istype(target, /mob/living/silicon/) && !istype(target, /mob/living/silicon/ai))
+    user.drop_item(src, target)
+    to_chat(user, "<span class='notice'>You install \the [src] on \the [target.name]'s surface.</span>")
+    active = 1
+    camera_target = target
+    rc_cameras += src
+    return
+/*
+    END REMOTE CYBORG CAMERA SECTION
+*/
+
+
+/*
+    START CONTROL BOARD SECTION
+*/
+/obj/item/device/syndicate_remote_cyborg_control_board
+  name = "R.C. Circuit Board"
+  desc = "A off-putting looking board. Instead of standard green and yellow, it is black with red circuits, there is a small antenna on the side."
+  icon = 'icons/obj/RemoteCyborg.dmi'
+  icon_state = "syndie_board_01"
+  w_class = W_CLASS_SMALL
+  flags = FPRINT | NOBLUDGEON
+  var/frequency = "syndicate"
+  var/active = 0
+  var/mob/living/silicon/cyborg
+  var/inUse
+  var/controller = null
+
+/obj/item/device/syndicate_remote_cyborg_control_board/afterattack(atom/target, mob/user as mob, proximity_flag)
+  if(!istype(target, /obj/item/robot_parts/robot_suit))
+    return
+  var/obj/item/robot_parts/robot_suit/robot_suit = target
+  var/turf/T = get_turf(robot_suit)
+  if(robot_suit.check_completion())
+    if(!istype(T,/turf))
+      to_chat(user, "<span class='warning'>You can't put the motherboard in, the frame has to be standing on the ground to be perfectly precise.</span>")
+      return
+
+    var/mob/living/silicon/robot/remote_control_robot/robot = new /mob/living/silicon/robot/remote_control_robot(get_turf(target), unfinished = 1)
+    robot.invisibility = 0
+    robot.custom_name = robot_suit.created_name
```

Comment:
Haven't looked at the original code but this looks like copypasta, if so, you should make a finish_borg_frame() proc